data/monster/Bosses/the_horned_fox.xml

```diff
@@ -39,7 +39,7 @@
 	<summons maxSummons="6">
 		<summon name="minotaur archer" interval="1000" chance="13" max="2" />
 		<summon name="minotaur guard" interval="1000" chance="13" max="2" />
```

Comment:
Not really, I just missed them. I am using basic search functions in sublime text as I don't understand much of regular expressions, so I miss some things. I'll try to find more occurrences of it.

By the way, I will be replacing melee attributes "attack" and "skill" with "min" and "max" in a few minutes.